src/health.js

```diff
@@ -18,10 +18,11 @@ router.get("/live", (_, res) => res.status(200).json({ status: "ok" }));
  */
 router.get("/ready", async (req, res, next) => {
   const isHealthy = await readinessCheck();
-    if (isHealthy) {
-      res.status(200).json({ status: "ok" });
-    }
+  if (isHealthy) {
+    res.status(200).json({ status: "ok" });
```

Comment:
just do
```
res.sendStatus(200)
```

Reply:
also instead of if {} else {} do if {return res.xxx} return res.xxxx

Reply:
```
if (isHealthy) {
    return res.sendStatus(200);
}
return res.status(503).send("Service Unavailable")
```

Reply:
Done